tests/integration/test_storage_rabbitmq/test.py

```diff
@@ -110,7 +110,7 @@ def rabbitmq_setup_teardown():
     ],
 )
 def test_rabbitmq_select(rabbitmq_cluster, secure):
-    if secure and instance.is_built_with_memory_sanitizer():
+    if secure and instance.is_built_with_thread_sanitizer():
```

Comment:
Not `if secure and (instance.is_built_with_memory_sanitizer() or instance.is_built_with_thread_sanitizer()):`?

Reply:
There were no MSAN warnings before, only TSAN, I used it mistakenly